
       ▓██▓▄
    ▓▓▓▓██▓█▓▄
     ████████▓▒
          ▀▓▓███▄      ▄▄   ▄               ▌
         ▄▌▌▓▓████▄    ██ ▓█▀  ▄▌▀▄  ▓▓▌▄   ▓█  ▄▌▓▓▌▄ ▌▌   ▓
       ▓█████████▌▓▓   ██▓█▄  ▓█▄▓▓ ▐█▌  ██ ▓█  █▌  ██  █▌ █▓
      ▓▓▓▓▀▀▀▀▓▓▓▓▓▓▌  ██  █▓  ▓▌▄▄ ▐█▓▄▓█▀ █▓█ ▀█▄▄█▀   █▓█
       ▓▌                           ▐█▌                   █▌
        ▓
  
version: 2.1.0-alpha23

🐰 Keploy: 2024-06-27T16:20:53+05:30 	[34mINFO[0m	config file not found; proceeding with flags only
🐰 Keploy: 2024-06-27T16:20:53+05:30 	[34mINFO[0m	GitHub Actions workflow file generated successfully	{"path": "/githubactions/keploy.yml"}
🐰 Keploy: 2024-06-27T16:20:53+05:30 	[34mINFO[0m	keploy initialized and probes added to the kernel.
🐰 Keploy: 2024-06-27T16:20:53+05:30 	[34mINFO[0m	Java detected and CA already exists	{"path": "/usr/lib/jvm/java-11-openjdk-amd64/lib/security/cacerts"}
🐰 Keploy: 2024-06-27T16:20:53+05:30 	[34mINFO[0m	Java detected and CA already exists	{"path": "/usr/lib/jvm/java-11-openjdk-amd64/lib/security/cacerts"}
🐰 Keploy: 2024-06-27T16:20:54+05:30 	[34mINFO[0m	Keploy has taken control of the DNS resolution mechanism, your application may misbehave if you have provided wrong domain name in your application code.
🐰 Keploy: 2024-06-27T16:20:54+05:30 	[34mINFO[0m	Proxy started at port:16789
🐰 Keploy: 2024-06-27T16:20:54+05:30 	[34mINFO[0m	starting TCP DNS server at addr :26789
🐰 Keploy: 2024-06-27T16:20:54+05:30 	[34mINFO[0m	starting UDP DNS server at addr :26789
 * Serving Flask app 'app'
 * Debug mode: on
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:6000
 * Running on http://192.168.1.8:6000
Press CTRL+C to quit
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 136-811-912
Signal received, canceling context...
Traceback (most recent call last):
  File "/home/akash/Desktop/Keploy/samples-python/flask-mongo/app.py", line 42, in <module>
    app.run(host='0.0.0.0', port=6000, debug=True)
  File "/home/akash/.local/lib/python3.10/site-packages/flask/app.py", line 1191, in run
    run_simple(t.cast(str, host), port, self, **options)
  File "/home/akash/.local/lib/python3.10/site-packages/werkzeug/serving.py", line 1090, in run_simple
    run_with_reloader(
KeyboardInterrupt
🐰 Keploy: 2024-06-27T16:20:57+05:30 	[34mINFO[0m	proxy stopped...
🐰 Keploy: 2024-06-27T16:20:58+05:30 	[34mINFO[0m	eBPF resources released successfully...
